lib/multi.c

```diff
@@ -1274,30 +1248,32 @@ static CURLMcode Curl_multi_wait(struct Curl_multi *multi,
 #endif
 
   if(nfds) {
```

Comment:
This condition needs to be updated to also be true if just the WinSock event is being waited on.